### PR TITLE
Publicize: add linkedin preview

### PR DESCRIPTION
**Pull request by jhnstn**
This adds Linkedin to publicize preview.
**Testing**
- Enable publicize sharing on `calypso.localhost:3000` : `ENABLE_FEATURES=publicize-scheduling make run`
- Authorize Linkedin on a premium site
- Open the sharing tab on a published post
- Click 'Preview'
- Linkedin should be available for previewing
- Clicking "Linkedin share" on the left should show the correct linkedin share preview

_breaks up #12644_

### CODE REVIEW

**Comment by jhnstn** [comment]
Copied comment from @iamtakashi : 

---

**LinkedIn**
* I don't see the account name.
* Can we show the company name of the account that is required info of an account?
* We need to show the site URL.

image

File: client/components/vertical-menu/items/social-item.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import SocialLogo from 'social-logos';
 const services = translate => ( {
 	facebook: { icon: 'facebook', label: translate( 'Facebook feed' ) },
 	google: { icon: 'google', label: translate( 'Google search' ) },
+	google_plus: { icon: 'google-plus', label: translate( 'Google+ ' ) },
```
Comment (a8ci18n) [inline client/components/vertical-menu/items/social-item.jsx]: Hi! I've found a possible matching string that has already been translated **55** times:
`translate( 'Google' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/publicize-linkedin-preview)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jhnstn** [comment]
@iamtakashi Adding the company name is bit tricky given the way access linkedin account information. Here is what we have now in this PR:

screen shot 2017-05-22 at 3 51 09 pm

**Comment by rralian** [comment]
FWIW, the comment here (https://github.com/Automattic/wp-calypso/pull/14198#issuecomment-302236857) is not how I see shares show up in Linkedin. Next to the avatar I see, three lines:

> **[name]**
[company]
[age of post]

In any case, let's merge this for now and we can tweak the design in a followup. I've tested and it works as designed.

**Comment by jhnstn** [comment]
Thanks @rralian

**Pull request merged**